blockchain.go

```diff
@@ -1620,13 +1613,13 @@ func (b *Blockchain) EndDebugging() {
 }
 
 func (b *Blockchain) CoverageReport() *runtime.CoverageReport {
-	return b.coverageReportedRuntime.CoverageReport
+	return b.coverageReport
 }
 
 func (b *Blockchain) SetCoverageReport(coverageReport *runtime.CoverageReport) {
-	b.coverageReportedRuntime.CoverageReport = coverageReport
+	b.coverageReport = coverageReport
 }
```

Comment:
I think this is pointless now, as we move with config

Reply:
Good catch! I've removed it entirely.